Editor/Auditors/BuildAuditor.cs

```diff
@@ -0,0 +1,139 @@
+        {
+#if UNITY_2019_4_OR_NEWER
+            var buildReport = GetBuildReport();
+            if (buildReport != null)
+            {
+                foreach (var packedAsset in buildReport.packedAssets)
+                {
+                    var dict = new Dictionary<GUID, List<PackedAssetInfo>>();
+                    foreach (var content in packedAsset.contents)
+                    {
+                        var assetPath = content.sourceAssetPath;
+                        if (!Path.HasExtension(assetPath))
+                            continue;
+
+                        if (Path.GetExtension(assetPath).Equals(".cs"))
+                            continue;
+
+                        if (!dict.ContainsKey(content.sourceAssetGUID))
+                        {
+                            dict.Add(content.sourceAssetGUID, new List<PackedAssetInfo>());
+                        }
+                        dict[content.sourceAssetGUID].Add(content);
+                    }
+
+                    foreach (var entry in dict)
+                    {
+                        var content = entry.Value[0];
+                        var assetPath = content.sourceAssetPath;
+
+                        ulong sum = 0;
+                        foreach (var v in entry.Value)
+                        {
+                            sum += v.packedSize;
+                        }
+
+                        var assetName = Path.GetFileNameWithoutExtension(assetPath);
+                        string description;
+                        if (entry.Value.Count > 1)
+                            description = string.Format("{0} ({1})", assetName, entry.Value.Count);
+                        else
+                            description = assetName;
+                        var issue = new ProjectIssue(k_Descriptor, description, IssueCategory.BuildFiles, new Location(assetPath));
+                        issue.SetCustomProperties(new[]
+                        {
+                            sum.ToString(),
+                            packedAsset.shortPath
+                        });
+                        onIssueFound(issue);
+                    }
+                }
+            }
+#endif
+            onComplete();
+        }
+
+        public static BuildReport GetBuildReport()
+        {
+            if (s_BuildReport != null)
+                return s_BuildReport;
+
+            if (!Directory.Exists(s_BuildReportDir))
+                Directory.CreateDirectory(s_BuildReportDir);
+
+            var date = File.GetLastWriteTime(s_LastBuildReportPath);
+            var assetPath = s_BuildReportDir + "/Build_" + date.ToString("yyyy-MM-dd-HH-mm-ss") + ".buildreport";
+
+            if (!File.Exists(assetPath))
+            {
+                File.Copy("Library/LastBuild.buildreport", assetPath, true);
+                AssetDatabase.ImportAsset(assetPath);
```

Comment:
Should this stay in the project as an asset or get cleaned up after analysis?

Reply:
Good question. Build Report Inspector does that so you can _inspect_ previously saved reports. Since this is not possible in Project Auditor, perhaps you are right, we should clean things up.

Reply:
I looked at deleting the asset and there is a problem: If we delete it, the _BuildReport_ object will be released from memory. Since the BuildReport summary info is accessed every frame, we would end up re-creating and deleting the asset every frame. I think I would leave it as is for the time being. I will revisit it as part of the load/save feature.